application/ui/components/endpoint.jsx

```diff
@@ -299,10 +300,13 @@ module.exports = React.createClass({
 
         if (this.state.status === LOADED)
         {
+            var latency = this.state.responseTimestamp > 0 ? (parseInt(this.state.responseTimestamp) - parseInt(this.state.requestTimestamp)) : 0;
             apiCallInfo = (
                 <ApiCallInfo status={this.state.status}
                              request={this.state.requestInfo}
-                             response={this.state.response} />
+                             response={this.state.response}
+                             latency={latency}
```

Comment:
We try to align these, with a space on either side of the `=`. Looks like that wasn't done here originally.